src/object/firefly.cpp

```diff
@@ -148,4 +149,11 @@ Firefly::get_settings()
   return result;
 }
 
+void
+Firefly::on_flip(float height)
+{
+  MovingSprite::on_flip(height);
+  FlipLevelTransformer::transform_flip(m_flip);
+}
+
```

Comment:
I'm starting to wonder; how feasible and maintanable would it be to use a `virtual bool is_flippable() const` inside MovingSprite objects?

Reply:
It would certainly be an option, but I'm not sure how readable that would be since some objects that are flippable would need that bool set to false if they need to handle the flipping differently (like objects where the flippability changes based on whether they are set to obey gravity or not, or Ispy, which changes directions between UP and DOWN instead of flipping, but it's still flippable).

The bool itself could be added (for possible use elsewhere in the future), but I'm not sure whether it'd be a good idea to use it to do the flip transforming inside the `MovingSprite` class - then the problem above happens and it wouldn't be obvious what the bool exactly means since flipping isn't a simple "yes flip" or "no flip" situation.